lua/complementree/utils.lua

```diff
@@ -9,4 +9,12 @@ function M.cword(complete_item)
   return (complete_item.word or complete_item.abbr)
 end
 
+function M.make_relative_path(path, root)
```

Comment:
This assumes that `root` is a prefix of `path`. Could you add a check for that using `vim.startswith` and error out if it is not the case ?